lib/task/import/csvCheckImportTask.class.php

```diff
@@ -137,6 +137,25 @@ protected function getDbConnection()
 
     protected function setCsvValidatorFilenames($filenameString)
     {
+        // If a directory's provided return an array of file paths
+        if (is_dir($filenameString)) {
+            $files = array_diff(scandir($filenameString), ['..', '.']);
+
+            $filePaths = [];
+
+            foreach ($files as $file) {
+                $filePath = $filenameString.DIRECTORY_SEPARATOR.$file;
+
+                if (is_file($filePath)) {
```

Comment:
I think it would be good to check `is_readable($filePath)` as well.  

I also looked at `CsvImportValidator::validateFileName($filePath);` as a possible alternative, but it throws an exception if any of the checks fail and doesn't check `is_file()` specifically.

Reply:
Sounds good!